### PR TITLE
Fix coding style issues

### PR DESCRIPTION
**Pull request by wezhang**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/HDInsightHelperImpl.java [hunk 1]
```diff
@@ -79,47 +78,53 @@ public String getInstallationId() {
...
 
-    private void openItem(@NotNull final Project project, @NotNull final VirtualFile virtualFile, @Nullable final VirtualFile closeableVirtualFile) {
-        ApplicationManager.getApplication().invokeLater(new Runnable() {
-            @Override
-            public void run() {
-                if(closeableVirtualFile != null) {
-                    FileEditorManager.getInstance(project).closeFile(closeableVirtualFile);
-                }
-                FileEditorManager.getInstance(project).openFile((VirtualFile) virtualFile, true, false);
+    private void openItem(@NotNull final Project project,
```
Comment (t-rufang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/HDInsightHelperImpl.java]: Maybe we can remove all the @NotNull anotation in this file.

Comment (wezhang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/HDInsightHelperImpl.java]: It's out of coding style issue scope...We can fix them when touching these codes in development.

Comment (t-rufang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/HDInsightHelperImpl.java]: But seems like you add some `Nullable` annotation to the existing code like the following one.
```java
    @Nullable
    private static VirtualFile getOpenedItem(final Project project) {
        final FileEditorManager fileEditorManager = FileEditorManager.getInstance(project);
        for (final VirtualFile editedFile : fileEditorManager.getOpenFiles()) {
            final IClusterDetail detail = editedFile.getUserData(JobViewEditorProvider.JOB_VIEW_KEY);
```

Comment (wezhang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/HDInsightHelperImpl.java]: Oh, no...

Comment (wezhang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/HDInsightHelperImpl.java]: Reverted.

**Pull request merged**